frontend/config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "baseURL": "https://api-dot-nav-e-387904.uc.r.appspot.com/"
+    "baseURL": "https://api-dot-nav-e-387904.uc.r.appspot.com/",
+    "testURL": "http://localhost:8080/"
```

Comment:
Lol the whole point of the config file is you can have different config for local. Just change baseurl locally and deploy the real url